Client/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -1109,6 +1109,41 @@ void MixedReadMaterialString(CScriptArgReader& argStream, CClientMaterial*& pMat
     }
 }
 
+//
+// Check 4x4 lua table
+//
+bool IsValidMatrixLuaTable(lua_State* luaVM, uint uiArgIndex)
+{
+    uint uiRow = 0;
+    uint uiCell = 0;
+
+    if (lua_type(luaVM, uiArgIndex) == LUA_TTABLE)
+    {
+        for (lua_pushnil(luaVM); lua_next(luaVM, uiArgIndex) != 0; lua_pop(luaVM, 1), uiRow++)
```

Comment:
what is that weird for loop? you can initialize `uiRow` (btw dont use hungarian notation) in the loop directly if needed